.solhint.json

```diff
@@ -9,6 +9,7 @@
     "named-parameters-mapping": "warn",
     "no-console": "off",
     "not-rely-on-time": "off",
-    "one-contract-per-file": "off"
+    "one-contract-per-file": "off",
+    "gas-custom-errors": "off"
```

Comment:
I understand why this one is necessary right now, but I created an issue to re-introduce this one as soon as this is fixed in solhint: https://github.com/cosmos/solidity-ibc-eureka/issues/109